### PR TITLE
Require Python 3.6+

### PR DESCRIPTION
**Pull request by dpgrote**
The version of pimcistandard is specified, currently to 0.06. This allows keeping track of the version of the picmistandard that should be used with each version of WarpX.

It also specifies that Python version >= 3.6 is required.

### CODE REVIEW

**Comment by ax3l** [comment]
Looks great, can we please document this in `Docs/source/building/python.rst` as well?

**Comment by dpgrote** [comment]
Done. I added the same comment in Docs/source/running_python/how_to_run.rst since it can never hurt to repeat things.

**Comment by MaxThevenet** [comment]
@atmyers would this work on Battra? If not, let is make sure Battra has Python >3.6 before merging this PR.

**Comment by atmyers** [comment]
I updated the default Python on battra last week to `3.6.10`.

File: Docs/source/building/python.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Installing WarpX as a Python package
 ====================================
 
-A full Python installation of WarpX can be done, which includes a build of all of the C++ code, or a pure Python version can be made which only installs the Python scripts.
+A full Python installation of WarpX can be done, which includes a build of all of the C++ code, or a pure Python version can be made which only installs the Python scripts. WarpX requires Pythone version 3.6 or newer.
```
[the remark this answers is not in the document]
Comment (ax3l) [reply Docs/source/building/python.rst:6]: (can be a follow-up fix, just forgot to commit this review)

**Pull request merged**